include/picongpu/param/atomicPhysics.param

```diff
@@ -0,0 +1,44 @@
+/* Copyright 2019-2020 Brian Marre
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ /** @file
+  * configuration file for atomicPhysics, see speciesDefinition.param for
+  * actual modelling of atomic Physics of ions.
+  */
+
+#pragma once
+
+#include <picongpu/particles/atomicPhysics/stateRepresentation/LevelVector.hpp>
+#include <picongpu/particles/atomicPhysics/stateRepresentation/ConfigNumber.hpp>
+
+namespace picongpu
+{
+namespace atomicPhysics
+{
+
+    //datatype/coding used in representation atomic state
+
+    template< uint8_t T_NumberStates >
+    using LevelVector = particles::atomicPhysics::stateRepresentation::LevelVector<
```

Comment:
you can add this to the `flylite.param` config - this is the project name of the folder you are reading ;)

Reply:
I wanted to keep the naming scheme in order to not confuse somebody later.
Since it is part of the atomicPhysics stage I would look for a atomicPhysics.param if I am searching for something having to do with this.
While it is thematically correct to add it to flylite.param, a user would have to know the connection between flylite and atomicPhysics to search for it there and I wanted to avoid this for the user.